tests/test_type_aliases.py

```diff
@@ -0,0 +1,30 @@
+from typing import Dict, List
+
+import pytest
+
+from stable_baselines3.common.type_aliases import check_cast, non_null
+
+
+def test_non_null():
+    for a in (1, "a", [2]):
+        assert non_null(a) == a
+
+    with pytest.raises(ValueError):
+        non_null(None)
+
+
+def test_check_cast():
+    assert check_cast(dict, {}) == {}
+    assert check_cast(dict[str, int], {}) == {}
+    assert check_cast(Dict[str, int], {}) == {}
+
+    with pytest.raises(TypeError):
+        check_cast(list[int], {})
+        check_cast(List[int], {})
```

Comment:
what's the point of this? I find this to be a false guarantee. this function doesn't actually check that the list contains ints, or that the dict above is str -> int.

Reply:
it's better than `# type: ignore` or `typing.cast(list[int], {})`, which is the alternative.

Reply:
This at least checks the base class.